python/vowpalwabbit/sklearn_vw.py

```diff
@@ -122,112 +124,189 @@ def __init__(self,
 
         Parameters
         ----------
-        probabilities
-        random_seed (int): seed random number generator
-        ring_size (int): size of example ring
-        convert_to_vw (bool): flag to convert X input to vw format
+
+        probabilities : float
+            Float number between 0 and 1
+        random_seed : integer
+            seed random number generator
+        ring_size : integer
+            size of example ring
+        convert_to_vw : bool
+            flag to convert X input to vw format
 
         Update options
-        bfgs: use L-BFGS optimization algorithm
-        mem: set the rank of the inverse hessian approximation used by bfgs
-        ftrl: use FTRL-Proximal optimization algorithm
-        ftrl_alpha: ftrl alpha parameter
-        ftrl_beta: ftrl beta parameter
-        learning_rate,l (float): Set learning rate
-        power_t (float): t power value
-        decay_learning_rate (float): Set Decay factor for learning_rate between passes
-        initial_t (float): initial t value
-        feature_mask (str): Use existing regressor to determine which parameters may be updated.
-                            If no initial_regressor given, also used for initial weights.
+
+        bfgs : use L-BFGS optimization algorithm
```

Comment:
Can you add the missing types for these?

Reply:
Yes, I will add them